lib/version.js

```diff
@@ -1,2 +1,2 @@
 // generated by genversion
-module.exports = '4.1.2-postman.1'
```

Comment:
Why do we have this in the diff? 🤔

Reply:
@coditva The master pipeline was failing after the last release. This fixes that failure.